Dockerfile-linux.template

```diff
@@ -333,6 +333,15 @@ RUN set -eux; \
 		--with-readline \
 		--with-zlib \
 		\
+{{ if env.variant == "cli" then ( -}}
+# https://github.com/docker-library/php/pull/1259
+		--enable-phpdbg \
+		--enable-phpdbg-readline \
```

Comment:
I'm not sure how difficult it would be to make the default of this mirror whether we did `--with-readline` but that's certainly above my pay grade! :smile: